presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
```

Comment:
hmm. this takes a looooong time to parse. i generated a 1M element array that was about a 7mb char[] and it took 1m24s